### PR TITLE
Shayan/add color to chart

### PR DESCRIPTION
**Pull request by shayanypn**

### CODE REVIEW

File: src/Pages/Stat/index.tsx [hunk 1]
```diff
@@ -104,15 +159,38 @@ const Stat: React.FC<IProps> = ({ match, history }) => {
     const data = assets
       .filter((item:any) => (item.name.indexOf('.exe.blockmap') === -1 && nameFilter(item.name)))
```
Comment (moshfeu) [inline src/Pages/Stat/index.tsx:160]: Why `any`? 😭

File: src/Pages/Stat/index.tsx [hunk 2]
```diff
@@ -104,15 +159,38 @@ const Stat: React.FC<IProps> = ({ match, history }) => {
...
+          item.download_count,
+          (extension.color || '#34595A')
+        ];
+      });
+
+
+    setChartOption(prevStat => ({
+      ...prevStat,
+      xAxis: {
+        categories: data.map((item:any) => item[0])
```
Comment (moshfeu) [inline src/Pages/Stat/index.tsx:175]: Consider 

```suggestion
        categories: data.map([item]) => item)
```

File: src/Pages/Stat/index.tsx [hunk 3]
```diff
@@ -104,15 +159,38 @@ const Stat: React.FC<IProps> = ({ match, history }) => {
...
+      },
+      series: [{
+        type: 'column',
+        name: 'Download',
+        data: data.map((item:any) => ({
```
Comment (moshfeu) [inline src/Pages/Stat/index.tsx:180]: Consider
```suggestion
        data: data.map(([y, color]) => ({
          y,
          color
```

File: src/Pages/Stat/index.tsx [hunk 4]
```diff
@@ -104,15 +159,38 @@ const Stat: React.FC<IProps> = ({ match, history }) => {
     const data = assets
       .filter((item:any) => (item.name.indexOf('.exe.blockmap') === -1 && nameFilter(item.name)))
       .map((item:any) => ({...item, ext: nameFilter(item.name)}))
-      .map((item:any) => ([item.ext, item.download_count]));
+      .map((item:any) => {
```
Comment (moshfeu) [inline src/Pages/Stat/index.tsx:162]: Why 2 `map`s?

**Comment by shayanypn** [comment]
@moshfeu just cleaned up the PR

File: src/Pages/Stat/index.tsx [hunk 5]
```diff
@@ -103,16 +158,35 @@ const Stat: React.FC<IProps> = ({ match, history }) => {
 
     const data = assets
       .filter((item:any) => (item.name.indexOf('.exe.blockmap') === -1 && nameFilter(item.name)))
```
Comment (moshfeu) [inline src/Pages/Stat/index.tsx:160]:
```suggestion
      .filter((item:any) => (!item.name.includes('.exe.blockmap') && nameFilter(item.name)))
```

Comment (shayanypn) [reply src/Pages/Stat/index.tsx]: @moshfeu cool, let do it.

**Pull request merged**